docs/Vue.js/whatIsVue.md

```diff
@@ -0,0 +1,31 @@
+---
+id: Vue.js Introduction
```

Comment:
change `id` in good format because we are using `id` for unique and for routing like `url` so, write in right way for SEO and best practice